src/CVNBot/ListManager.cs

```diff
@@ -163,9 +179,15 @@ public string AddUserToList(string name, string project, UserType type, string a
                 // User was originally unlisted or is on non-conflicting list
                 using (IDbCommand cmd = dbcon.CreateCommand())
                 {
-                    cmd.CommandText = "INSERT INTO users (name, project, type, adder, reason, expiry) VALUES ('" + name.Replace("'", "''")
-                    + "','" + project + "','" + ((int)type).ToString() + "','" + adder.Replace("'", "''")
-                    + "','" + reason.Replace("'", "''") + "','" + GetExpiryDate(expiry) + "')";
+                    cmd.CommandText = "INSERT INTO users (name, project, type, adder, reason, expiry) VALUES (@name,@project,@type,@adder,@reason,@expiry)";
+                    cmd.Parameters.Add(new SqliteParameter("@name", name));
+                    cmd.Parameters.Add(new SqliteParameter("@project", project));
+                    cmd.Parameters.Add(new SqliteParameter("@type", ((int)originalType).ToString()));
```

Comment:
This is inserting `originalType` instead of `type`, this means when adding a user to the database (e.g. `wl add` or `bl add`) it is persisting the unknown original implied state of "unlisted" instead of actually inserting the specified type!